### PR TITLE
fix: Updated non-bot-check to pass for bot

### PR DESCRIPTION
**Pull request by alexandrtovmach**

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
@indirect you right, please set to one required approval =)

File: .github/workflows/non-bot-check.yml [hunk 1]
```diff
@@ -27,41 +27,44 @@ jobs:
...
-              ref: merge_commit_sha,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-            });
-            const protectedFiles = files.filter(({ filename }) => /^spec\//.test(filename))
-            if (protectedFiles && protectedFiles.length) {
-              await github.issues.createComment({
-                issue_number: context.issue.number,
+            console.log(`PR created by ${login}`);
+            if (login !== 'github-actions') {
```
Comment (alexandrtovmach) [inline .github/workflows/non-bot-check.yml]:
```suggestion
            if (login !== 'github-actions' || login !== 'github-actions[bot]') {
```

**Review by indirect — APPROVED** [review]
@alexandrtovmach reduced to one approval 👍

**Pull request merged**